yarn.lock

```diff
@@ -4992,16 +4992,16 @@ __metadata:
     single-call-balance-checker-abi: "npm:^1.0.0"
     uuid: "npm:^8.3.2"
   peerDependencies:
-    "@metamask/accounts-controller": ^24.0.0
+    "@metamask/accounts-controller": ^26.0.0
     "@metamask/approval-controller": ^7.0.0
-    "@metamask/keyring-controller": ^19.0.0
+    "@metamask/keyring-controller": ^21.0.0
     "@metamask/network-controller": ^22.0.0
     "@metamask/permission-controller": ^11.0.0
-    "@metamask/preferences-controller": ^15.0.0
+    "@metamask/preferences-controller": ^17.0.0
```

Comment:
We should probably update the `preferences-controller` too now. I've checked and the last 2 majors are just because of a peer dep bump (for the `keyring-controller`).

Could you try bumping it to `^17.0.0`? This way we would get rid of this warning:
```
➤ YN0060: │ @metamask/preferences-controller is listed by your project with version 15.0.2 (p47d41), which doesn't satisfy what @metamask/assets-controllers requests (^17.0.0).
```